.github/workflows/update_dependabot_prs.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Dependency Pr Update
+on:
+  pull_request:
+    types: [labeled, opened, reopened]
+
+jobs:
+  pr-check:
+    name: Update Dependabot Prs
+    if:  contains(github.event.pull_request.labels.*.name, 'dependencies') && contains(github.event.pull_request.labels.*.name, 'component:ui')
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+
+      - name: Update PR Body
+        env:
+            GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+            OWNER: ${{ github.repository_owner }}
+            REPO: ${{ github.event.repository.name }}
+            BRANCH: ${{github.event.pull_request.head.ref}}
+        run: |
+          gh pr checkout ${{ env.BRANCH }}
+          gh pr edit --body "Bug, Docs Fix or other nominal change"
```

Comment:
It looks like this is just going to replace the body per https://cli.github.com/manual/gh_pr_edit

But dependabot seems to put some potentially useful stuff in the body i.e. https://github.com/ansible/awx/pull/12606

Should we see if we can do like a `gh pr view` to get the body and then append our string to it and set the new body as the result?